app-misc/irtrans-irserver/irtrans-irserver-6.09.04-r1.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic toolchain-funcs mono-env multilib
+
+RESTRICT="strip"
+
+DESCRIPTION="Server software for IRTrans"
+HOMEPAGE="http://www.irtrans.de"
+SRC_URI="http://ftp.disconnected-by-peer.at/irtrans/irserver-src-${PV}.tar.gz
+	 http://ftp.disconnected-by-peer.at/irtrans/irserver-${PV}.tar.gz
+	http://www.irtrans.de/download/Server/Linux/irserver-src.tar.gz -> irserver-src-${PV}.tar.gz
+	http://www.irtrans.de/download/Server/Linux/irserver.tar.gz -> irserver-${PV}.tar.gz"
+
+LICENSE="BSD GPL-2"
+SLOT="0"
+KEYWORDS="~x86 ~amd64 ~arm"
+IUSE="mono"
+
+RDEPEND="mono? ( >=dev-lang/mono-2.10.5 )"
+
+S="${WORKDIR}"
+
+src_prepare() {
+	sed -e 's!^ODIRARM = .*!ODIRARM = n800!' -i makefile || die
+	default
+}
+
+src_compile() {
+	append-flags -DLINUX -DMEDIACENTER
+	append-ldflags --static
+
+	# Set sane defaults (arm target has no -D flags added)
+	local irbuild=irserver_arm_noccf
+	irserver=irserver
+
+	# change variable by need
+	if use x86 ; then
+		irbuild=irserver
+	elif use amd64 ; then
+		irbuild=irserver64
+		irserver=irserver64
+	elif use arm ; then
+		irbuild=irserver_arm
+	fi
+
+	emake CXX="$(tc-getCXX)" CC="$(tc-getCC)" CFLAGS="${CFLAGS}" \
+		CXXFLAGS="${CXXFLAGS}" LDFLAGS="${LDFLAGS}" "${irbuild}"
+}
+
+src_install() {
+	newbin "${WORKDIR}/${irserver}" irserver
+
+	keepdir /etc/irserver/remotes
+
+	docinto remotes
+	dodoc remotes/*
+
+	newinitd "${FILESDIR}"/irtrans-server.initd irtrans-server
+	newconfd "${FILESDIR}"/irtrans-server.confd irtrans-server
+
+	if use mono ; then
```

Comment:
`use mono &&`

Reply:
I'm not sure how i should use this here. Using ```use mono &&``` is useful for oneliners. But since we do a few different things i would need to use ```use mono &&``` multiple times. Isn't a simply ```if use mono; then ... fi``` better here?